### ISSUE TITLE
victory should be available via cdn

### ISSUE THREAD

**Issue by boygirl** [FormidableLabs/victory#11]

**Comment by jakepusateri** [comment]
[npmcdn.com](npmcdn.com) allows this very easily

https://npmcdn.com/victory@0.0.2/dist/victory.min.js

**Comment by boygirl** [comment]
Add npmcdn urls to all victory component repos

**Comment by exogen** [comment]
As far as I know, the expected usage of the `dist` bundle looks something like:

```html
<div id="app"></div>
<script src="//fb.me/react-0.14.3.js"></script>
<script src="//fb.me/react-dom-0.14.3.js"></script>
<script src="//npmcdn.com/victory-bar@2.5.0/dist/victory-bar.js"></script>
<script>
  var bar = React.createElement(VictoryBar);
  ReactDOM.render(bar, document.getElementById("app"));
</script>
```

...which doesn't work: there's an error in Radium's `enhancer.js`:

```
TypeError: Cannot read property 'Component' of undefined
```

Something in Radium or VictoryBar is not using the global React, which most likely means that our webpack config in `builder-victory-component` is messed up.

**Comment by knowbody** [comment]
fixed by #134 

you can also check JSBin or JSFiddle examples and play around with Victory:
- https://jsfiddle.net/5zrx7wxk/1/
- http://jsbin.com/cenopi/edit?html,js,output

**Comment by escherize** [comment]
Also, cljsfiddle now! http://cljsfiddle.com/#gist=76289b7500a39349051bef8a2ad3aced
